pkg/controller/backup_controller.go

```diff
@@ -251,7 +264,10 @@ func (controller *backupController) processBackup(key string) error {
 	// update status
 	updatedBackup, err := controller.client.Backups(ns).Update(backup)
 	if err != nil {
-		glog.V(4).Infof("error updating status to %s: %v", backup.Status.Phase, err)
+		logContext.WithFields(logrus.Fields{
+			"phase": backup.Status.Phase,
+			"error": err,
+		}).Info("error updating Backup status")
```

Comment:
maybe I don't understand how logrus works, but shouldn't this be s/Info/Error

As above, please don't log and return the error.